docs/components/input.md

```diff
@@ -33,6 +33,21 @@ If the label is omitted it can be handled with another component, like **KLabel*
 </template>
 ```
 
+### labelAttributes
+
+Use the `labelAttributes` prop to configure the the **KLabel's** props if using the `label` prop.
```

Comment:
Can you add a link to the KLabel > props section?

Reply:
Turns out we don't have a props section in `KLabel` so I just linked to `KLabel`